notebooks/ImvizExample.ipynb

```diff
@@ -333,7 +332,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.imshow(mask.to_image(data.shape), origin='lower')"
+    "plt.imshow(mask.to_image(data.data.shape), origin='lower')"
```

Comment:
Why the extra `.data`?

Reply:
Without the extra `.data` I get `AttributeError: 'NDData' object has no attribute 'shape'` with these images.

Reply:
Well, that's weird. It is supposed to be glue Data object. What has changed while I was away?

Reply:
p.s. NDData having no ndim might be a design oversight. Even CCDData has ndim. I should ask astropy.